README.md

```diff
@@ -296,7 +282,6 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Space Shark Room](http://liooil.neocities.org/VR_webStuff/room+07/room07.html) - Hinduist shark cage in space
 - [Metal Gear REX](http://www.jmvisualcreativity.es/vr_metal/) - Return to Shadow Moses. SNAKE!
 - [Meditation Orbs](http://www.skyislandsvr.com/pages/meditationorbs.html) - A place to meditate
-- [Welcome](http://drawvr.com/welcome/) - To A-Frame
```

Comment:
Sorry that was a mistake! I thought I had seen that before and removed that as a duplicate. Will send another PR after I go through remaining list too. And correct this one and the one you noted above (https->http)